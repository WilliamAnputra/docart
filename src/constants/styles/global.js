// RED CATEGORIES
export const LIGHT_RED = '#e33e41';
export const SEMI_LIGHT_RED = '#D12C12';
export const MEDIUM_RED = '#f20800';
export const SEMI_DARK_RED = '#ce2124';
export const DARK_RED = '#a91515';

// BLACK CATEGORIES
export const BLACK = '#000000';
export const LIGHT_BLACK = '#2D2D2D';

// BLUE CATEGORIES
export const LIGHT_BLUE = '#4A90E2';
export const DARK_BLUE = '#253f61';

// GRAY CATEGORIES
export const LIGHT_GRAY = '#cccccc';
export const SEMI_MEDIUM_GRAY = '#d2dae0';
export const MEDIUM_GRAY = '#9b9b9b';

export const DARK_GRAY = '#4a4a4a';

// GREEN CATEGORIES
export const LIGHT_GREEN = '#95B487';
export const MEDIUM_GREEN = '#96b35c';

// PURPLE CATEGORIES
export const MEDIUM_PURPLE = '#C7008C';

// ORANGE CATEGORIES
export const LIGHT_ORANGE = '#FF9E7D';
export const DARK_ORANGE = '#d95a5c';

// WHITE CATEGORIES
export const WHITE = '#ffffff';
export const MEDIUM_WHITE = '#f4f4f4';

// YELLOW CATEGORIES
export const MEDIUM_YELLOW = '#F5A623';
