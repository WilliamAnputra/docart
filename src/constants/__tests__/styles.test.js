import React from 'react';

import * as globalColors from '../styles/global';

it('Renders colors', () => {
    expect(globalColors).toMatchSnapshot();
});

