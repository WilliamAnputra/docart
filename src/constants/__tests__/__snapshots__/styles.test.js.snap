// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders colors 1`] = `
Object {
  "BLACK": "#000000",
  "DARK_BLUE": "#253f61",
  "DARK_GRAY": "#4a4a4a",
  "DARK_ORANGE": "#d95a5c",
  "DARK_RED": "#a91515",
  "LIGHT_BLACK": "#2D2D2D",
  "LIGHT_BLUE": "#4A90E2",
  "LIGHT_GRAY": "#cccccc",
  "LIGHT_GREEN": "#95B487",
  "LIGHT_ORANGE": "#FF9E7D",
  "LIGHT_RED": "#e33e41",
  "MEDIUM_GRAY": "#9b9b9b",
  "MEDIUM_GREEN": "#96b35c",
  "MEDIUM_PURPLE": "#C7008C",
  "MEDIUM_RED": "#f20800",
  "MEDIUM_WHITE": "#f4f4f4",
  "MEDIUM_YELLOW": "#F5A623",
  "SEMI_DARK_RED": "#ce2124",
  "SEMI_LIGHT_RED": "#D12C12",
  "SEMI_MEDIUM_GRAY": "#d2dae0",
  "WHITE": "#ffffff",
}
`;
